vendor/github.com/linode/linodego/nodebalancer_configs.go

```diff
@@ -281,7 +281,7 @@ func (c *Client) UpdateNodeBalancerConfig(ctx context.Context, nodebalancerID in
 	if bodyData, err := json.Marshal(updateOpts); err == nil {
 		body = string(bodyData)
 	} else {
-		return nil, NewError(err)
+		return nil, NewError(fmt.Errorf("Error converting NodeBalancer update options to JSON %v", err))
```

Comment:
Can we yank this change to the vendor file?
The equivalent should be set at cloud/linode/loadbalancers_test.go:721 if the error description is wanted.